packages/@ourworldindata/grapher/src/schema/grapher-schema.003.yaml

```diff
@@ -482,7 +485,9 @@ properties:
                             default: true
                             description: Whether to render this time series in the table sheet
                         shortUnit:
-                            type: string
+                            type:
+                                - string
+                                - "null"
```

Comment:
This one we can probably quickly migrate, I don't think we should have explicit `null`s in there.

Reply:
Yeah you're right of course. I thought that finding where this keeps leaking into our db might be a bit tricky but I had a look in the ETL and quickly found the offending code and just submitted a PR over there